### DISCUSSION TITLE
Opt out of feeds, explore repositories, latest changes, etc. completely.

### DISCUSSION THREAD

**Discussion by innocentius** [community/community#57318] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think there needs to be a way for users to opt out of this whole dashboard mess completely. You are adding in more annoying features everyday, and we don't want to see it.

The best way to impliment this is to create an option for users to be redirected to their own profile page when visiting github.com, instead of your social media bullshit.

Who is with me?

**Comment by alatarum** [comment]
I also would like a way to redefine startpage. I don't need this useless dashboard which promotes something that I didn't ask and I can't tune for my needs.

**Comment by branch-tim** [comment]
The new dashboard is a huge downgrade.  It was bad before, and now less useful.  Almost the only things I do in the UI are PRs and look for repositories.  Both of those are inconvenient and tedious.
